mdagent/tools/base_tools/analysis_tools/rdf_tool.py

```diff
@@ -0,0 +1,203 @@
+from typing import List, Optional
+
+import matplotlib.pyplot as plt
+import mdtraj as md
+from langchain.tools import BaseTool
+from pydantic import BaseModel, Field
+
+from mdagent.utils import PathRegistry
+
+
+class RDFToolInput(BaseModel):
+    trajectory_fileid: str = Field(
+        None, description="Trajectory file. Either dcd, hdf5, xtc oe xyz"
+    )
+
+    topology_fileid: Optional[str] = Field(None, description="Topology file")
+    stride: Optional[int] = Field(None, description="Stride for reading trajectory")
+    selections: Optional[List[List[str]]] = Field(
+        [["protein", "water"]],
+        description="Selections for RDF. Do not use for now. As "
+        "it will only calculate RDF for protein and water molecules.",
+    )
+    # atom_indices: Optional[List[int]] = Field(
+    #    None, description="Atom indices to load in the trajectory"
+    # )
+    # TODO: Add pairs of atoms to calculate RDF within the tool
+    ##pairs: Optional[str] = Field(None, description="Pairs of atoms to calculate RDF ")
+
+
+class RDFutils:
```

Comment:
this doesn't appear to be used anywhere - can we remove unused code and extra comments?